docs/user/README.md

```diff
@@ -159,6 +159,45 @@ nodes:
   replicas: 2
 ```
 
+kind also supports multiple control-plane and HA setups using a stacked etcd topology:
+```yaml
+# six node cluster with three control-plane and three worker nodes
+kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+nodes:
+- role: control-plane
+  replicas: 3
+- role: worker
+  replicas: 3
+- role: external-load-balancer
+```
+
+The `external-load-balancer` node is [HAProxy][haproxy] based and it acts like a load
+balancer in front of the control-plane nodes. This node has to be added explicitly.
+
+There are also plans to support an external etcd cluster topology.
+You can read more about the HA topologies that kubeadm supports [here][kubeadm-ha].
+
+Next to `role` and `replicas` you can also specify fields like `image` to be able to
+use a custom node image per node:
+```yaml
+# a setup with custom control-plane and worker node images
+kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+nodes:
+- role: control-plane
+  image: mycontrolplane-node-image:latest
+- role: worker
+  image: myworker-node-image:latest
+  replicas: 2
+```
+
+The `Node` object also allows you to patch the kubeadm config that the node uses by applying
```

Comment:
nit: I think there should be a more explicit mention of what the `Node` object is. This reference kind of appears out of nowhere.
Maybe right below the first config file example add something like "each entry in 'nodes' represents a [`Node` object](ref to godoc)."